popup.js

```diff
@@ -67,18 +69,94 @@ function outputHTMLString(messages) {
  *     scripts.
  * @param {number} i The index of messages to use.
  */
-function fillRemainingHoles(messages, i) {
+function fillRemainingHolesAndMinimizeStyles(messages, i) {
   var html = messages[i].html;
   var frameHoles = messages[i].frameHoles;
   for (var index in frameHoles) {
     if (frameHoles.hasOwnProperty(index)) {
       var frameIndex = frameHoles[index];
       for (var j = 0; j < messages.length; j++) {
         if (messages[j].frameIndex == frameIndex) {
-          fillRemainingHoles(messages, j);
+          fillRemainingHolesAndMinimizeStyles(messages, j);
```

Comment:
Yes, it does miss those elements.  It's because document.getElementById isn't returning them.  For some reason, even though their ids are specified in the html, the document that is rendered does not have ids for those elements.  I think it might have to do with where I place the style elements (I place the style tags for pseudo elements and fonts before the html and head elements), but I am looking into it.